afu_gefahrenkartierung_pub_export_ai/build.gradle

```diff
@@ -27,4 +48,71 @@ task pubGefkart(type: Db2Db){
 		new TransferSet('pub/ueberflutungskarte.sql', 'afu_gefahrenkartierung_pub.gefahrenkartirung_ueberflutungskarte', true),
 		new TransferSet('pub/ik_synoptisch_mgdm.sql', 'afu_gefahrenkartierung_pub.gefahrenkartirung_ik_synoptisch_mgdm', true)			
 	];    
-}
\ No newline at end of file
+}
+
+/*
+2. Transform the data from model SO_AfU_Gefahrenkartierung into bafu model Hazard_Mapping using GRETL sql
+*/
+task mapGefkart2Mgdm(type: Db2Db, dependsOn: 'pubGefkart'){
```

Comment:
Hierfür wäre eigentlich ein SqlExecutor besser, da in der selben DB. Oder gibt es einen bestimmten Grund für den Db2Db-Step?

Reply:
Wie besprochen - werde ich aufgrund des Testaufwandes belassen